frontend/src/v4/routes/viewerGui/components/issues/issues.component.tsx

```diff
@@ -18,7 +18,6 @@
 import { PureComponent } from 'react';
```

Comment:
Isn't missing "for approval" ? 

This is in the original issue #4584 
![image](https://github.com/3drepo/3drepo.io/assets/11945337/42a1d4f4-ca87-44d2-8bfd-91bff82426d5)

And this is in the PR:
<img width="372" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/b4925a4e-a81e-4a34-bb7b-8111a03b2dbc">